.ci/jenkins/dsl/jobs.groovy

```diff
@@ -258,6 +213,23 @@ KogitoJobUtils.createVersionUpdateToolsJob(this, 'optaplanner', 'Drools', [
   properties: [ 'version.org.drools' ],
 ])
 
+void setupSpecificNightlyJob(Folder specificNightlyFolder) {
```

Comment:
If I get it right, while the existing `Jenkinsfile.nightly` is a high-level pipeline that triggers the `Jenkinsfile.deploy`, this new "specific" pipelines just test the single repository with native, Quarkus, and Mandrel.

Reply:
yep. This `Jenkinsfile.deploy` has much more options that we really don't need onto the specific nightlies.
In the end, I would like to move this all to the build-chain at some point as we are repeating a lot of things between nightlies and PRs...